apiserver/observer/recorder.go

```diff
@@ -16,20 +16,26 @@ import (
 // recorders that that will update the observer and the auditlog
 // recorder when it records a request or reply. The auditlog recorder
 // can be nil.
-func NewRecorderFactory(observerFactory rpc.ObserverFactory, recorder *auditlog.Recorder) rpc.RecorderFactory {
+func NewRecorderFactory(
+	captureArgs bool,
```

Comment:
Personal preference, but this arg IMO should go to the end of the param list.
So (factory, record, capture)
It looks icky to see callers go NewRecorderFactory(true, factory, recorder)
Also, we should define consts for RecordArgs and NoRecordArgs and use those instead of true/false

Reply:
Ok, thanks!